src/commands/china.ts

```diff
@@ -0,0 +1,30 @@
+import { ApplyOptions } from "@sapphire/decorators";
+import type { ApplicationCommandRegistry } from "@sapphire/framework";
+import { Time } from "@sapphire/time-utilities";
+import { CommandInteraction, MessageEmbed } from "discord.js";
```

Comment:
We are now using discord.js v14 which means MessageEmbed is now EmbedBuilder.
You also need Colors for embed colors now.
Lastly CommandInteraction is now ChatInputCommandInteraction

```suggestion
import { ChatInputCommandInteraction, EmbedBuilder, Colors } from "discord.js";
```